tests/integration/route-test.js

```diff
@@ -0,0 +1,71 @@
+import Ember from 'ember';
+import startApp from '../helpers/start-app';
+import { test, moduleFor } from 'ember-qunit';
+
+let App;
+
+const indexURL  = '/';
+
+const firstTab = 'button:nth-child(1)';
+const secondtTab = 'button:nth-child(2)';
+const thirdTab = 'button:nth-child(3)';
+const fourthTab = 'button:nth-child(4)';
+
+const firstContent = '#new-york';
+const secondContent = '#los-angeles';
+const thirdContent = '#chicago';
+const fourthContent = '#san-francisco';
+
+moduleFor('route:application', 'integration:application', {
+  beforeEach() {
+    App = startApp();
+  },
+
+  afterEach() {
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('tabs function as expected', function(assert) {
+  assert.expect(24);
+
+  return visit(indexURL).then(() => {
+    assert.equal(find(firstContent).text(), 'Financial');
+    assert.equal(find(secondContent).text(), 'Hollywood');
+    assert.equal(find(thirdContent).text(), 'Chiraq');
+    assert.equal(find(fourthContent).text(), 'Tech');
+
+    assert.equal(find(firstContent).attr('style'), undefined, 'displays correctly by user-provided default value');
+    assert.equal(find(secondContent).attr('style'), 'display: none;', 'content is hidden');
```

Comment:
This pattern tests implementation rather than behavior. You should be able to use `.is(':visible')` and `.is(':hidden')`.